src/store/coinListStore/CategoryStore.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  action,
+  makeAutoObservable,
+  makeObservable,
+  observable,
+  override,
+  runInAction,
+} from "mobx";
+
+import {
+  CategoryListModel,
+  CategoryModel,
+  normalizeCategoryModel,
+} from "store/models/coinList/category";
+import { Meta } from "utils/meta";
+import { ILocalStore } from "utils/useLocalStore";
+
+import { requestCategoryList } from "./requestCoinList";
+
+export default class CategoryStore implements ILocalStore {
+  _meta: Meta = Meta.initial;
+  options: CategoryModel[] = [];
+  _value: CategoryModel = {
```

Comment:
```suggestion
  private _meta: Meta = Meta.initial;
  options: CategoryModel[] = [];
  private _value: CategoryModel = {
```
сделай их приватными раз уж назвал с нижнего подчеркивания и сделал для них геттеры